### PR TITLE
portals: Generate portals based on body name.

### PR DESCRIPTION
**Pull request by aero31aero**

### CODE REVIEW

File: routes/components/portals.js [hunk 1]
```diff
@@ -25,21 +25,43 @@ router.get('/', authenticate, elevate, function (req, res, next) {
 	}
 });
 
-router.get('/:body', authenticate, elevate, function (req, res, next) {
+router.get('/bodies/:body', authenticate, elevate, function (req, res, next) {
```
Comment (rogeredthat) [inline routes/components/portals.js]: Why does it have to go through this extra subroute?

Comment (aero31aero) [inline routes/components/portals.js]: To not conflict with `/portals/accom` or other potential portals that are not a body.

Comment (rogeredthat) [inline routes/components/portals.js]: Plan on making another sub for non-event entities as well?

Comment (aero31aero) [inline routes/components/portals.js]: Yes. `/portals/bodies/rome/events/gladiator-fights` down to the last level. Eventually.

**Pull request merged**